var/spack/repos/builtin/packages/cxx/test/hello.c++

```diff
@@ -0,0 +1,10 @@
+#include <stdio.h>
+
+int main()
+{
+
+	printf ("Hello world from C++\n");
+	printf ("YES!");
+
+	return (0);
```

Comment:
why (0)

Reply:
I'm assuming it's there because some compiler failed on that in the past